native/cocos/core/assets/EffectAsset.h

```diff
@@ -544,6 +544,12 @@ struct IShaderInfo {
         if (version == "glsl4") return &glsl4;
         return nullptr;
     }
+    IShaderSource *getSource(const ccstd::string &version) {
```

Comment:
add const version

Reply:
It is a non-const version. Will modify it outside?

Reply:
Yes, NativeProgramLibrary will overwrite bindings of shader source.